gnd/src/main/java/com/google/android/gnd/ui/common/BindingAdapters.java

```diff
@@ -49,6 +50,12 @@ public static void bindGoogleSignOnButtonClick(
     button.setOnClickListener(onClickCallback);
   }
 
+  @BindingAdapter("onButtonClick")
+  public static void bindButtonClick(
```

Comment:
A new binding adapter should be needed; please use listener bindings instead:
https://developer.android.com/topic/libraries/data-binding/expressions#listener_bindings

Reply:
updated in latest commit https://github.com/google/ground-android/pull/845/commits/3c961768928f7035287435e791a1b195ff37ae0c of #845